kpagacz/wiki-on-fire
# add: added examples of requests to WikiMedia API

Closes #11

Quite surprisingly there is no authentication requred to use WikiMedia API as long as we follow the guidelines and don't bombard them with hundreds of requests per second. 

Test:
npm install and node requests.js

You can have a look at the responses to get a glimpse of what the responses look like. Might be useful for further database design. I probably should transform those into a couple of functions. For now there are only 2 endpoints as well here. It's a good start.

I think the next steps should include:
* transform those get requests into a function parametrized by date
* research lambdas
* set up storage for the data (might be our database, might be something else)
* download the data from those endpoint into the storage

Would also be very nice to have a way to seed our database with the wiki data. I am proposing this because this is not a lot of data, it's historic data, so it won't change and we want to minimize the number of requests sent to WikiMedia API. So we might want to look into how to nicely convert the results into seed files (with possible intermediate steps?).
